src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -2649,7 +2654,7 @@ private IAsyncResult BeginExecuteReaderInternal(CommandBehavior behavior, AsyncC
                                             // back into pool when we should not.
                 }
 
-                bool usedCache;
+                bool usedCache = false;
```

Comment:
Is this change necessary? the default value is false.

Reply:
On the netcore side, if the assignment is left out, I get a compile error at the call to TriggerInternalEndAndRetryIfNecessary, where it is referenced. (`Error CS0165 Use of unassigned local variable 'usedCache'`) I don't get the error on the netfx side, but made the change in both places to keep the code in sync.